aspnetcore/mvc/models/file-uploads.md

```diff
@@ -430,28 +430,33 @@ if (string.IsNullOrEmpty(ext) || !permittedExtensions.Contains(ext))
 
 ### File signature validation
 
-A file's signature is determined by the first few bytes that make up the file. These bytes can be used to indicate if the extension matches the content of the file. The sample app checks file signatures for common file types. In the following example, the file signature for a Microsoft Word document is checked against the file:
+A file's signature is determined by the first few bytes that make up the file. These bytes can be used to indicate if the extension matches the content of the file. The sample app checks file signatures for for a few common file types. In the following example, the file signature for a JPEG image is checked against the file:
```

Comment:
I think you mean `for a few common file types.` and not `for for a few common file types.`

Reply:
oooooops. 🙈 I'll fix that on the next commit. Thx.

Reply:
This was caused by a common problem working on docs: When using a wide window and a line breaks across the double-word, they're *very easy* to miss!